zenodo/modules/records/config.py

```diff
@@ -99,6 +101,10 @@
     ('isCompiledBy', _('Compiled by')),
     ('isDerivedFrom', _('Derived from')),
     ('isSourceOf', _('Source of')),
+    ('requires', _('Requires')),
+    ('isRequiredBy', _('Required material')),
```

Comment:
```suggestion
    ('isRequiredBy', _('Required by')),
```

Just for consistency with the rest of the terms I guess